Android.mk

```diff
@@ -59,8 +59,14 @@ OPENTHREAD_COMMON_FLAGS                                          += \
     $(NULL)
 endif
 
-ifneq ($(USE_OTBR_DAEMON), 1)
-OPENTHREAD_COMMON_FLAGS  += -DOPENTHREAD_CONFIG_UDP_FORWARD_ENABLE=1
+ifeq ($(USE_OTBR_DAEMON), 1)
+OPENTHREAD_COMMON_FLAGS                                          += \
```

Comment:
It seems we're both adding these flags
https://github.com/openthread/openthread/pull/4568.
So maybe I can close my PR?

Reply:
No, your PR adds default value, there will be no duplicate definitions.